cmd/initialize/init_policies.go

```diff
@@ -0,0 +1,102 @@
+// Copyright 2022 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+// Define the `ec init-policies` command
+package initialize
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	hd "github.com/MakeNowJust/heredoc"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	"github.com/enterprise-contract/ec-cli/internal/utils"
+)
+
+func initCmd() *cobra.Command {
```

Comment:
NIT: this is a bit odd because `init` is a command on its own. Following the convention we have used in the other commands here, consider naming this `initPoliciesCmd`. It would make this line in `init.go` a little bit easier to read: `InitCmd.AddCommand(initCmd())` -> `InitCmd.AddCommand(initPoliciesCmd())`